common.mk

```diff
@@ -9,6 +9,10 @@ else
 $(info Running with RISCV=$(RISCV))
 endif
 
+ifndef CONDA_DEFAULT_ENV
```

Comment:
Can this check be merged with the existing check for RISCV in this makefile? If both RISCV and CONDA are undefined, error.

Reply:
My impression was that some people wanted to not use conda hence the warning instead of erroring.

Reply:
No I mean combine with this: https://github.com/ucb-bar/chipyard/blob/b00da55a870ea5429cf32968a47a3e5db297ac4a/common.mk#L6-L10

If `RISCV` is not defined, throw an error (as before), except change the error message to ask the user if they did the Conda stuff specifically.

If `RISCV` is defined, don't throw any error or warning.

Reply:
Ok, so you don't want a conda check. Instead, just check RISCV and just change the error message to point to conda. Correct?

Reply:
Yeah